gulpfile.js

```diff
@@ -10,11 +14,34 @@ gulp.task("build:backend:assets:copy-js-vendors", function() {
       .pipe(gulp.dest("./js/vendors"));
 });
 
+gulp.task("build:backend:sass:generate-css", function() {
+  return gulp.src([
+    "./src/Backend/Core/Layout/Sass/screen.scss",
+    "./src/Backend/Core/Layout/Sass/debug.scss",
+  ])
+      .pipe(sourcemaps.init())
+      .pipe(sass({
+        includePaths: [
+          "./node_modules/"
```

Comment:
Personally, I found this more confusing, as you have to "know" there are multiple include-paths.